default.nix

```diff
@@ -32,9 +32,15 @@ let
     llvmPackages_12.bintools
     rustc-nightly
     cargo-nightly
-    xargo
     wasmtime
     rust-bindgen
+    python3
+    tree
+  ] ++ pkgs.lib.optional pkgs.stdenv.isDarwin [
+    libiconv
+    # The following are usually also necessary:
+    # pkgs.darwin.apple_sdk.frameworks.Security
+    # pkgs.darwin.apple_sdk.frameworks.CoreFoundation
```

Comment:
Well, are they necessary or are they not necessary? :)

Reply:
nix team said these are often necessary so wanted to leave a hint here. Removed now.